code-of-conduct.md

```diff
@@ -0,0 +1,66 @@
+# Code of Conduct
+
+- This document, like all Codes of Conduct, is a standard for good behavior.
+- We (the staff of Helix Launcher) have drafted it to provide accountability -- to help users understand what behavior is and isn't welcome.
+- **People who don't read this document will still be subjected to it. If you don't understand a relevant part of it, feel free to ask us or other community members for clarification.**
+- We want this document to be clear, readable, and concise. [Right now it is none of those things ;w;]
+
+## [Draft notes – write your own when redrafting; remove them in the final draft]
+
+- This draft will sound dry, aggressive and terse. Most of my serious writing does.
+- I deal in absolutes when needed. (i'm not a sith cktvgbypnmi,o)
+- I don't expect or want this draft to sound anything like the final one.
+- Should we use a google doc or smth (preferably something owned by a company that respects its users instead)
+
+## Rules
+
+[These should be unambiguous and specific.]
+
+## Guidelines
+
+While you are here, we expect you to treat other users well, regardless of outside social expectations. Misunderstandings **will** happen and that's okay.
+
+Here are some ways you can treat others well:
+
+- Be kind and respectful.
+- Report users who break the rules or guidelines.
+- Take responsibility for your mistakes, and learn from them.
+- If someone wants you to leave them alone, leave them alone.
+
+### Bigotry
+Here are some common forms of bigotry in online spaces. We detest them ^^
+- Homophobia
+- Transphobia
+- Neuronormativity
```

Comment:
I'm not sure this list is a good idea, at least in this form. It's not exhaustive, but your wording makes it sound exhaustive.
I'd more suggest something along the lines of (but not word-by-word) "We do not accept bigotry in our spaces", perhaps with "including but not limited to" if you do think that list is necessary.